docs/testing/smoke-testing.md

```diff
@@ -0,0 +1,134 @@
+# Smoke Testing
+
+We generally consider smoke testing using this definition [from Wikipedia](https://href.li/?https://en.wikipedia.org/wiki/Smoke_testing_(software)):
+
+> **Smoke Testing** is a subset of test cases that cover the most important functionality of a component or system, used to aid assessment of whether the main functions of the software appear to work correctly. It is a set of tests run on each new build of a product to verify that the build is testable before the build is released into the hands of the test team
+
+When testing builds the following things should be tested to ensure critical parts of the Blocks plugin are still functional.
+
+## Setup
+
+To make future testing more efficient, we recommend setting up some Blocks in advance so you can repeat tests on them whenever smoke testing.
+
+### 1. Create a page with all regular and SSR blocks (such as the product grids) setup and configured.
+
+<details>
+<summary>You can copy and paste the following code into a new page to add all the blocks (click):</summary>
```

Comment:
Pasting this should trigger the parser and would behave the same way as if you opened an existing page.